src/solvers.jl

```diff
@@ -468,7 +468,7 @@ function prepConstrMatrix(m::Model)
     A = rowmat'
 end
 
-function vartypes_without_fixed(m::Model)
+function cats_without_fixed(m::Model)
```

Comment:
Maybe ``varcats`` instead of ``cats``? Otherwise I'm happy with ``vartypes`` since we're talking about translating into MPB format.